BREAKING_CHANGES.md

```diff
@@ -0,0 +1,8 @@
+# Breaking Change
+
```

Comment:
I think adding the following will be a good idea as we do more breaking changes in the future.

```suggestion

## v1.x to v2.x
```